build.gradle

```diff
@@ -120,8 +120,10 @@ dependencies {
   compile group: 'org.springframework', name: 'spring-context-support', version: '4.3.10.RELEASE'
```

Comment:
There are too many refactors in this PR making it too big to safely review,
it would be appreciated if you could pull some of these out to separate PRs.

i.e. StaffTemplate changes, introducing a DocumentService, changing CCJ etc to use claimService